### PR TITLE
Add Additional Styling To Form

### PR DESCRIPTION
**Pull request by sampoder**
These are slightly opinionated changes but I added some extra styling that makes the form closer to other forms / sites we have... thoughts?

### CODE REVIEW

File: app/views/hackathons/submissions/new.html.erb [hunk 1]
```diff
@@ -1,25 +1,25 @@
-<% content_for :title, "Apply to list your hackathon" %>
+<% content_for :title, "List your hackathon on hackathons.hackclub.com" %>
```
Comment (sampoder) [inline app/views/hackathons/submissions/new.html.erb:1]: feels weird to be applying, as long as you meet the requirements your in... idk i might be the one who cares about this sorta thing

File: app/views/hackathons/submissions/new.html.erb [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
-  <div class="mb4">
+  <div class="mb2">
     <p>
       If you'd like to list your hackathon on
       <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>,
       please fill out this form! We will get back to you within a couple days.
     </p>
 
-    <p class="mb0">
```
Comment (sampoder) [inline app/views/hackathons/submissions/new.html.erb:17]: i thought that this made the whole thing more readable and more consistent

File: app/views/hackathons/submissions/new.html.erb (hunk 1 above)
Comment (northeastprince) [reply app/views/hackathons/submissions/new.html.erb:1]: I think it reflects the fact that it needs approval - you're not immediately on the site.

Comment (sampoder) [reply app/views/hackathons/submissions/new.html.erb:1]: I think we make that clear in the description, making it an application feels like this is some sort of exclusive list... at least that's my reading of the word

Comment (northeastprince) [reply app/views/hackathons/submissions/new.html.erb:1]: If it's below, then I'm fine with this.

File: app/views/hackathons/submissions/new.html.erb (hunk 2 above)
Comment (northeastprince) [reply app/views/hackathons/submissions/new.html.erb:17]: If we need some general spacing on the bottom, we should probably have that on the general container/layout.

File: app/views/hackathons/submissions/_form.html.erb [hunk 3]
```diff
@@ -75,7 +75,7 @@
           <%= sma.input :city, required: false %>
           <%= sma.input :province, label: "State/Province", required: false %>
           <%= sma.input :postal_code, label: "ZIP/Postal Code", required: false %>
-          <%= sma.input :country_code, as: :country, required: false %>
+          <%= sma.input :country_code, label: "Country", as: :country, required: false %>
```
Comment (garyhtou) [inline app/views/hackathons/submissions/_form.html.erb:78]: oof good catch!

File: app/assets/stylesheets/shared/_form.scss [hunk 4]
```diff
@@ -54,12 +55,30 @@ form.simple_form {
...
   }
+  
+  .form__inputs--nested > div > label {
+    font-size: 18px;
+    margin-bottom: var(--spacing-2);
+  }
+  
+  .form__inputs--nested > .lead {
+    font-size: var(--font-2)!important;
+  }
```
Comment (garyhtou) [inline app/assets/stylesheets/shared/_form.scss:70]: very nice!

File: app/assets/stylesheets/shared/_form.scss [hunk 5]
```diff
@@ -54,12 +55,30 @@ form.simple_form {
...
   }
+  
+  input::file-selector-button {
+    font-family: "Phantom Sans";
+    padding: 4px;
+    text-align: left;
+    background: var(--sunken);
+    border: 1px solid var(--border);
+    border-radius: var(--radii-small);
+  }
```
Comment (garyhtou) [inline app/assets/stylesheets/shared/_form.scss:83]: Would it be possible to also get rid of the odd left padding on the button?

image

File: app/views/hackathons/submissions/new.html.erb (hunk 1 above)
Comment (garyhtou) [reply app/views/hackathons/submissions/new.html.erb:1]: I'm cool with this!

File: app/views/hackathons/submissions/new.html.erb [hunk 6]
```diff
@@ -1,25 +1,25 @@
-<% content_for :title, "Apply to list your hackathon" %>
+<% content_for :title, "List your hackathon on hackathons.hackclub.com" %>
 <% image = "https://cloud-rmjrb2y89-hack-club-bot.vercel.app/0oy1pj1mznloa96gh1sg-oiontn0tkxelxwjutwobkza.jpeg" %>
 
 <%= render "shared/full_page_card", background_url: image do %>
   <%= image_tag "flag-standalone.svg", style: "max-width: 30%; margin-bottom: var(--spacing-3)" %>
   <h1>
-    Apply to list your hackathon
+    List your hackathon on <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>.
```
Comment (garyhtou) [inline app/views/hackathons/submissions/new.html.erb:7]: what are your thoughts on removing the trailing period?

also, overflow on mobile

image

**Review by garyhtou — APPROVED** [review]
i'm cool with how this is

i also left a couple nit-picky suggestions. feel free to address them, or not

File: app/assets/stylesheets/shared/_form.scss (hunk 5 above)
Comment (sampoder) [reply app/assets/stylesheets/shared/_form.scss:87]: let's do it!

File: app/views/hackathons/submissions/new.html.erb (hunk 6 above)
Comment (sampoder) [reply app/views/hackathons/submissions/new.html.erb:7]:
```suggestion
    List your hackathon on <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>
```

Let's do it!

File: app/views/hackathons/submissions/new.html.erb [hunk 7]
```diff
@@ -28,4 +28,10 @@
   </div>
 
   <%= render "hackathons/submissions/form" %>
+  <style>
+    h1 > a {
+      text-decoration: none;
+    }
+  </style>
```
Comment (sampoder) [inline app/views/hackathons/submissions/new.html.erb:35]:
```suggestion
```

File: app/views/hackathons/submissions/new.html.erb [hunk 8]
```diff
@@ -29,3 +29,4 @@
 
   <%= render "hackathons/submissions/form" %>
 <% end %>
+
```
Comment (sampoder) [inline app/views/hackathons/submissions/new.html.erb:32]:
```suggestion
```

**Comment by sampoder** [comment]
lgtm!

**Pull request merged**